paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormUI.kt

```diff
@@ -36,20 +40,23 @@ internal fun FormInternal(
     hiddenIdentifiersFlow: Flow<List<IdentifierSpec>>,
     enabledFlow: Flow<Boolean>,
     elementsFlow: Flow<List<FormElement>?>,
-    lastTextFieldIdentifierFlow: Flow<IdentifierSpec?>
+    lastTextFieldIdentifierFlow: Flow<IdentifierSpec?>,
+    modifier: Modifier = Modifier
 ) {
     FormUI(
-        hiddenIdentifiersFlow,
-        enabledFlow,
-        elementsFlow,
-        lastTextFieldIdentifierFlow
-    ) {
-        Loading()
-    }
+        hiddenIdentifiersFlow = hiddenIdentifiersFlow,
+        enabledFlow = enabledFlow,
+        elementsFlow = elementsFlow,
+        lastTextFieldIdentifierFlow = lastTextFieldIdentifierFlow,
+        loadingComposable = {
+            Loading()
+        },
+        modifier = modifier
+    )
 }
 
 @Composable
-private fun Loading() {
+internal fun Loading() {
```

Comment:
Why is this necessary?

Reply:
It's going to be used in `BaseAddPaymentMethodFragment`.